test/cql-pytest/test_describe.py

```diff
@@ -447,6 +447,75 @@ def test_desc_index(cql, test_keyspace):
         
         assert f"CREATE INDEX named_index ON {tbl}{maybe_space}(c)" in c_desc
 
+def test_desc_index_on_collections(cql, test_keyspace):
+    # In this test, all assertions are in form of
+    # `assert create_stmt in desc or create_stmt.replace("(", " (", 1)`
```

Comment:
Maybe I'm misreading something, but isn't the "or" part an error? Didn't you for get the "in desc" again? :-(

Reply:
Right, I missed that.